packages/block/package.json

```diff
@@ -58,10 +58,10 @@
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-implicit-dependencies": "^1.0.4",
     "istanbul": "^0.4.1",
-    "karma": "^5.2.1",
-    "karma-browserify": "^7.0.0",
+    "karma": "^6.3.2",
```

Comment:
Out of interest: have you looked if this has any practical consequences (performance, test quality or realism 😄 ,...)?

Reply:
i was updating the deps in trying to resolve some miscellaneous issues but it was all working fine so i decided to keep the version bumps :)